nsqd/channel.go

```diff
@@ -2493,7 +2493,7 @@ func (c *Channel) peekAndReqDelayedMessages(tnow int64, delayedQueue *DelayQueue
 						c.GetName(), tnow, m)
 				}
 
-				nsqMsgTracer.TraceSub(c.GetTopicName(), c.GetName(), "DELAY_QUEUE_TIMEOUT", m.TraceID, &m, "", 0)
+				nsqMsgTracer.TraceSub(c.GetTopicName(), c.GetName(), "DELAY_QUEUE_TIMEOUT", m.TraceID, &m, "", time.Now().UnixNano() - m.Timestamp)
```

Comment:
Maybe just use `tnow` here to avoid call `time.Now()` too much since `tnow` should be enough precise.

Reply:
ok. I will close the PR and open another one since some checks were not successful